proxy/src/source.rs

```diff
@@ -137,6 +137,17 @@ impl Source for Local {
     }
 
     fn register_project(&self, name: String, description: String, img_url: String) -> Project {
+        use futures::Future;
+        let client = oscoin_client::Client::new_from_file().unwrap();
+        let sender = client.new_account().wait().unwrap();
+        let project_address = oscoin_client::Address::zero();
+        let url = "https://example.com";
+
+        client
+            .register_project(sender, project_address, url.to_string())
+            .wait()
+            .unwrap();
+
```

Comment:
Lots of unwraps, can we instead return a `Result` from this function?

Reply:
Sure thang, this is just to show that we can do an end-to-end registration. Np proper care has been taken in rgards to ergonomics.